cmd/new.go

```diff
@@ -281,12 +276,17 @@ func newNewCmd() *cobra.Command {
 				fmt.Println(displayOpts.Color.Colorize(deployMsg))
 			}
 
+			if template.Quickstart != "" {
+				fmt.Println(template.Quickstart)
+			}
+
 			return nil
 		}),
 	}
 
-	cmd.PersistentFlags().StringVarP(&cloudURL,
-		"cloud-url", "c", "", "A cloud URL to download templates from")
+	cmd.PersistentFlags().StringArrayVarP(
```

Comment:
@lukehoban and I chatted about it yesterday, and we think so. He suggested `-s` as shorthand, though, it occurs to me now that that would conflict with `up`'s `-s` shorthand for `--stack` (which I want to add to `new` anyway to be able to specify the stack explicitly to avoid the prompt).

Some ideas:
 1. `--secret` and `-cs`
 1. `--configsecret` and `-cs`
 1. `--secret` and `-sc`
 1. `--secretconfig` and `-sc`

Any preference or other idea?

Reply:
I guess my preference would be to start with just `--secret` and if it ends up that we need a shorter option, come up with a good idea then?

I guess in general, settings these values "on the command line" is a bit of an anti-pattern anyway, since they will be logged into bash history by default?  So if you get prompted anyway, maybe best to leave it off for now?

Reply:
I decided to just hold off on `--secret` for now. For now, you'll have to enter these when prompted.